client/src/components/ScheduleGrid.tsx

```diff
@@ -57,14 +57,20 @@ let makerows = function (start: number, end: number) {
 function addCourse(rows: any, day: any, startTime: number, endTime: number, courseName: string) {
     if (startTime >= endTime || courseName === "" || courseName === null) return;
     let colnum = day + 1;
+<<<<<<< HEAD
     for (let row of rows) {
```

Comment:
Looks like these merge conflicts werent fixed properly indicated by the `<<<<<< HEAD` ...